AnimateQuotes.js

```diff
@@ -0,0 +1,91 @@
+import React, { useRef, useEffect, useState } from 'react';
+import { Animated, Text, View } from 'react-native';
+import { useIsShake } from './useIsShake';
+import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, SVG_IN_DELAY, SVG_IN_DURATION, SVG_OUT_DURATION, TRANSITION_TIME_2 } from './constants';
+
+export default function AnimateQuotes(props) {
+  const [isRegistered, setIsRegistered] = useState(false);
+  const [hasToggledBefore, setHasToggledBefore] = useState(false);
+
+  const fadeAnim = useRef(new Animated.Value(0)).current;
+  const sizeAnim = useRef(new Animated.Value(0.5)).current;
+
+  const { toggle } = useIsShake();
```

Comment:
Per our conversation earlier today, let's try to move all of the instantiations of `useShake` to a single source of truth higher in the app tree. Can be addressed in a follow up PR